docs/changehistory/NextVersion.md

```diff
@@ -84,3 +84,9 @@ It is now possible to retrieve `Units` from schemas stored in IModels. The new [
 
     UiFramework.setIModelConnection(iModelConnection, true);
 ```
+
+## AppUI Updates
+
+### WidgetState changes
+
+The property [WidgetDef.state]($appui-react) will now return `WidgetState.Closed` if the widget is in a panel that is collapsed, or the panel size is 0 or undefined. When `WidgetState.Open` is passed to the method [WidgetDef.setWidgetState] the containing panel will also open if it is in a collapsed state.
```

Comment:
`[WidgetDef.setWidgetState]` should be `[WidgetDef.setWidgetState]($appui-react)`.

Reply:
Fixed, thanks for catching.